.gitignore

```diff
@@ -1,2 +1,3 @@
 .astro/settings.json
 node_modules/
+.DS_Store
```

Comment:
Me several months ago would have objected to this, on grounds of it's just some random OS thing we should always keep out.

After several months of continuously accidentally adding `.DS_STORE`s: ... yeah. Might as well.

Reply:
Next time you start a project, this might be helpful: https://github.com/tejasq/add-gitignore

Reply:
Nifty!